ONSdigital/ras-maintenance-page
# Design system migration

# Motivation and Context
Move to the Design System, to remove unnecessary assets, and to adopt new styles.

# What has changed
* Base template introduced, based on design system standard
* Header refactored based on DS standard

* Body refactored based on DS standard
* Footer refactored based on DS standard
* Old classes removed

# How to test?
* Start server
* Confirm both views appear as intended.

# Links
* https://trello.com/c/7mZAVcz8
